.gitignore

```diff
@@ -10,3 +10,9 @@ dist
 .yarnrc
 *.egg-info
 .tox
+pip-wheel-metadata
```

Comment:
Aren't these changes supposed to be here? Shouldn't they disappear when rebasing on the proper underlying branch (be that master or br69)?

Reply:
They are here as it has been rebase on br69 but later changed to master. So I had to rebase it again on master